style.css

```diff
@@ -337,3 +432,410 @@ input:focus {
+    grid-column: 3;
+    grid-row: 2;
+    margin-top: -130px;
+  }
+
+  .work-card:nth-child(6) {
+    background: url(./assets/img/laptop\ 6.png);
+    background-position: top center;
+    background-repeat: no-repeat;
+    background-size: 100% 100%;
+    grid-column: 3;
+    grid-row: 1;
+    width: 100%;
+    height: 60%;
+  }
+
+  .work-card-grid .work-card:nth-child(1) {
+    margin-top: 126px;
+  }
+
+  .work-card:nth-child(4) .work-card-text {
+    display: none;
+  }
+
+  .work-card:nth-child(5) .work-card-text {
+    display: none;
+  }
+
+  .work-card:nth-child(6) .work-card-text {
+    margin-top: 60px;
+  }
+
+  .work-card:nth-child(6) .work-card-text h3 {
+    font-size: 25px;
+    overflow: hidden;
+    white-space: nowrap;
+    text-overflow: ellipsis;
+  }
+
+  .work-card-grid .work-card:nth-child(1) .work-card-text {
+    display: none;
+  }
+
+  .work-card:nth-child(6) .lang-container li:last-child {
+    overflow: hidden;
+    white-space: nowrap;
+    text-overflow: ellipsis;
+  }
+
+  .work-card:nth-child(6) .lang-container .lang_mobile {
+    display: none;
+  }
+
+  .work-card:nth-child(6) .lang-container .lang_desktop {
+    display: block;
+  }
+
+  .work-card .laptop {
+    display: none;
+  }
+
+  .work-card-grid {
+    display: grid;
+    grid-template-columns: 30% 40% 30%;
+    grid-template-rows: 1fr 1fr 1fr;
+    margin-top: 4px;
+  }
+
+  .image-geometry-1 {
+    display: block;
+  }
+
+  .image-geometry-2 {
+    margin-top: -314px;
+    display: block;
+    grid-column: 3 / 3;
+    grid-row: 3;
+    max-width: 100%;
+  }
+
+  .image-geometry-4 {
+    display: block;
+    width: 87.69px;
+    height: 87.71px;
+    grid-column: 2;
+    grid-row: 3;
+    justify-self: end;
+    margin-right: 178px;
+    margin-top: -50px;
+  }
+
+  .image-geometry-5 {
+    display: block;
+    width: 211.7px;
+    height: 124.04px;
+    grid-column: 1;
+    grid-row: 3;
+    margin-top: -50px;
+  }
+
+  .primary-btn-2 {
+    display: flex;
+    flex-direction: row;
+    background: #fff;
+    color: #1a2236;
+    margin-top: 40px;
+    width: 212px;
+  }
+
+  .primary-btn-2:hover {
+    background: #1a2236;
+    color: #fff;
+    margin-top: 40px;
+    width: 212px;
+  }
+
+  .right_arrow_black {
+    display: flex;
+    width: 50%;
+    height: 50%;
+    margin: 3px 0 0 10px;
+  }
+
+  .right_arrow {
+    display: none;
+  }
+
+  .about {
+    display: flex;
+    flex-direction: row;
+    gap: 60px;
+    margin: 0 80px;
+  }
+
+  .about-intro {
+    width: 30%;
+    order: 2;
+  }
+
+  .about-intro .primary-btn {
+    display: flex;
+    flex-direction: row;
+    width: 70%;
+  }
+
+  .about_cards {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows: 1fr 1fr;
+    gap: 0;
+  }
+
+  .about_cards .dots_img_teal {
+    display: block;
+    width: 80%;
+    height: 80%;
+    align-self: flex-end;
+    justify-self: end;
+    margin: 0 -60px -60px 0;
+  }
+
+  .get-started {
+    margin-top: 100px;
+  }
```

Comment:
- [ ] [OPTIONAL] Hey Biftu, Great job making the changes requested previously. It seems like the previous reviewer forgot to mention this but kindly add a `#f9f9fa` background color to the "Get started" section as specified in the [Figma design. 🎯](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23-9&t=t45uZqimwMP5BSMR-0)